### DISCUSSION TITLE
Can someone help me make a simple import case working for me ?

### DISCUSSION THREAD

**Discussion by ymolists** [cuelang/cue#1050] [Q&A] [answered]
I am trying to import a type from a package. I could not find an example that shows how i can achieve this in the docs. I looked at the docs in [Modules, Packages, and Instances](https://cuelang.org/docs/concepts/packages/) but was not able to figure this. Would it make sense to add a simple case ?

I am most likely missing something since i am new here so please bear with me 

```cue
// ./main.cue
package main

import (
  "example.com/pkg/test"
)

// A spec is of type #Spec
spec: #test.Spec
spec: {
  kind: "Homo Sapiens"
}
```

```cue
// ./test/test.cue
package test

#Spec: {
  kind: string
}
```

```cue
// ./cue.mod/module/module.cue 
module: "example.com/pkg"
```

```
$cue export 
package "test" not found:
    ./main.cue:4:3
spec: reference "#test" not found:
    ./main.cue:8:7
```

Edit:
```
$tree
.
├── cue.mod
│   └── module.cue
├── main.cue
└── test
    └── test.cue
```

**Comment by ymolists** [comment, marked as answer]
i changed it to `test.#Spec` instead and it works !

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/1050.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
